### PR TITLE
[#9886] App shows 'buy' for stickerpack user owns when pressing on st…

### LINKED ISSUE #9886: App shows 'buy' for stickerpack user owns when pressing on sticker in chat (Infura - related?)
# Bug Report

## Problem
App requires to buy stickerpack that was bought previously from "Tozemoon" sticker pack opened by tap on Tozemoon's sticker from inside chat view. 
NOTE: It's all fine when I open sticker market place from message input sticker icon. And also if I done that - there is no issue. Until I re-login to account.

So for me now, with RC4 - there are no problems when I open Tozemoon sticker pack from message input sticker icon
But issue is when I logged in to account (with bought but not installed Tozemoon) -> open chat with Tozemoon sticker -> tap tozemoon sticker from chat view 

#### Expected behavior
can install stickerpack

#### Actual behavior
photo_2020-01-22 15 06 35

### Reproduction

[comment]: # (Describe how we can replicate the bug step by step.)

- Open Status
- Restore account from seedphrase that bought 'TOZEMOON' before
- Join public chat with tozemoon sticker sent (#tozemoon)
- Tap upon Tozemoon sticker
**Actual result:** No Install button available but 20 SNT instead

### Additional Information

- Status version: nightly 22/01/2020
- Operating System: Android, iOS

### Logs
[stick-log.log](https://github.com/status-im/status-react/files/4097699/stick-log.log)

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #9886

### CODE REVIEW

File: src/status_im/wallet/core.cljs [hunk 1]
```diff
@@ -28,21 +28,13 @@
             [status-im.ui.components.bottom-sheet.core :as bottom-sheet]))
 
 (defn get-balance
-  [{:keys [address on-success on-error number-of-retries]
-    :as params
-    :or {number-of-retries 4}}]
-  (log/debug "[wallet] get-balance"
```
Comment (rasom) [inline src/status_im/wallet/core.cljs:34]: Do we have other logs instead? Previous time it helped to spot an issue in @Ferossgp local changes, so I would keep them as is

File: src/status_im/wallet/core.cljs [hunk 2]
```diff
@@ -183,15 +175,11 @@
       balances)))
 
 (defn get-token-balances
-  [{:keys [addresses tokens init? assets number-of-retries]
-    :as params
-    :or {number-of-retries 4}}]
-  (log/debug "[wallet] get-token-balances"
```
Comment (rasom) [inline src/status_im/wallet/core.cljs:189]: same here for logs

File: src/status_im/wallet/core.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/wallet/core.cljs:34]: ok i'll add them, thanks

**Review by rasom — APPROVED** [review]

**Review by yenda — APPROVED** [review]
add back logs otherwise looking great!

**Comment by churik** [comment]
Still can see price (20 SNT) instead of "Install" button if tap on TOZEMOON sticker in chat.
telegram-cloud-document-4-5933709422088947107

If I open stickerpack from the bottom panel - "Install" button is shown, and after that regardless of tapping from chat or from input field - only "Install" button is displayed, as expected.
Contact me for seed phrase of test user.

**Comment by flexsurfer** [comment]
thanks @churik , fixed!

**Pull request merged**